data_handler/database/models.py

```diff
@@ -34,3 +35,20 @@ class LoanState(Base):
     debt = Column(JSON)
     deposit = Column(JSON, nullable=True)
 
+
+class LiquidableDebt(Base):
+    """
+    SQLAlchemy model for the liquidable debt table.
+    """
+
+    __tablename__ = "liquidable_debt"
+
+    protocol = Column(ChoiceType(LendingProtocolNames, impl=String()), nullable=False)
+    user = Column(String, index=True, nullable=False)
+    liquidable_debt = Column(DECIMAL, nullable=False)
+    health_factor = Column(DECIMAL, nullable=False)
+    collateral = Column(JSON, nullable=False)
+    risk_adjusted_collateral = Column(DECIMAL, nullable=False)
+    debt = Column(JSON, nullable=False)
+    debt_usd = Column(DECIMAL, nullable=False)
```

Comment:
After you aggregate the data for all users of the given protocol, you can drop `user`, `health_factor`, `collateral`, `risk_adjusted_collateral` , `debt`, and `debt_usd`.